modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -468,6 +468,29 @@ TEST_F(ONNXtest, InferMultOutput)
     // Validate
     validate();
 }
+
+TEST_F(ONNXClassificationTest, InferPaddingCHW)
+{
+    // FIXME: Remove this test and add padding to test with FRCNN
+    useModel("classification/squeezenet/model/squeezenet1.0-9");
+    in_mat1 = initMatrixRandU(CV_8UC3, cv::Size{212, 200});
+
+    G_API_NET(SqueezNet, <cv::GMat(cv::GMat)>, "squeeznet");
+    cv::GMat in;
+    cv::GMat out = cv::gapi::infer<SqueezNet>(in);
+    cv::GComputation comp(cv::GIn(in), cv::GOut(out));
+
+    auto net = cv::gapi::onnx::Params<SqueezNet> { model_path }
+        .cfgMeanStd({ mean }, { std })
+        // Resize input image to (200, 199) and add padding that divisible to 32
+        // Should get (224, 224) input image for squeezenet1.0-9
+        // if input image size is incorrect than test failed
```

Comment:
It seems I don't understand the semantics for this function completely.

Is there an API document available?

Reply:
Removed after "padding" discussion.